templates/ubuntu20/hw/gpu.dockerfile.j2

```diff
@@ -15,6 +15,9 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 {% else %}
 WORKDIR ${INTEL_OPENVINO_DIR}/install_dependencies
-RUN ./install_NEO_OCL_driver.sh --no_numa -y -d ${INTEL_OPENCL} && \
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends software-properties-common && \
```

Comment:
I saw that Artym added installation such package in the script, we need only remove it

Reply:
Thanks!

Reply:
how about rhel images?

Reply:
Changes for RHEL are not required